Doc/howto/logging.rst

```diff
@@ -326,7 +326,7 @@ favourite beverage and carry on.
 If your logging needs are simple, then use the above examples to incorporate
 logging into your own scripts, and if you run into problems or don't
 understand something, please post a question on the comp.lang.python Usenet
-group (available at https://groups.google.com/forum/#!forum/comp.lang.python) and you
+group (available at `https://groups.google.com/forum/#!forum/comp.lang.python`_) and you
```

Comment:
There is no need for this to make the link a clickable link, see https://docs.python.org/3/howto/logging.html.

This syntax is usefull if you want to change the wording of the link like:

    `foo <https://>`_